src/controls/MyTeams/components/Team/Team.tsx

```diff
@@ -0,0 +1,283 @@
+    (async () => {
+      try {
+        dispatch({
+          type: ETeamTypes.SET_HAS_ERROR,
+          payload: false,
+        });
+        dispatch({
+          type: ETeamTypes.SET_IS_LOADING,
+          payload: true,
+        });
+        dispatch({
+          type: ETeamTypes.SET_MESSAGE,
+          payload: undefined,
+        });
+
+        const _members: ITeamMenber[] = await getTeamMembers(team.id);
+        const teamOwners: ITeamMenber[] = await getTeamOwners(team.id);
+        let _renderOwners: string[] = [];
+        for (const teamOwner of teamOwners) {
+          _renderOwners.push(teamOwner.displayName);
+        }
+        dispatch({
+          type: ETeamTypes.SET_TEAM_OWNERS,
+          payload: _renderOwners.join(","),
+        });
+
+        let publicChannels: IContextualMenuItem[] = [];
+        let privateChannels: IContextualMenuItem[] = [];
+        let _renderMembers: string[] = [];
+        let allChannels: IContextualMenuItem[];
+
+        for (const teamMember of _members) {
+          _renderMembers.push(teamMember.userId);
+        }
+        dispatch({
+          type: ETeamTypes.SET_TEAM_MEMBERS,
+          payload: _renderMembers,
+        });
+
+        // get Channels
+        const channels = await getTeamChannels(team.id);
+
+        publicChannels.push({
+          itemType: ContextualMenuItemType.Header,
+          key: "public",
+          text: "Public Channels",
+          iconProps: { iconName: "People" },
+        });
+        publicChannels.push({
+          itemType: ContextualMenuItemType.Divider,
+          key: "div",
+        });
+        for (const channel of channels) {
+          switch (channel.membershipType) {
+            // channels private
+            case EMembershipType.Private:
+              privateChannels.push({
+                iconProps: { iconName: "ChatInviteFriend" },
+                itemType: ContextualMenuItemType.Normal,
+                key: channel.id,
+                text: channel.displayName,
+                data: { webUrl: channel.webUrl, teamId: team.id },
+                secondaryText: channel.isFavoriteByDefault ? "isFavourite" : "",
+                onClick: onClickChannel,
+              });
+              break;
+            // Channel
+            default:
+              publicChannels.push({
+                iconProps: { iconName: "ChatInviteFriend" },
+                itemType: ContextualMenuItemType.Normal,
+                key: channel.id,
+                text: channel.displayName,
+                data: { webUrl: channel.webUrl, teamId: team.id },
+                secondaryText: channel.isFavoriteByDefault ? "isFavourite" : "",
+                onClick: onClickChannel,
+              });
+              break;
+          }
+        }
+        // at the end add private header
+        publicChannels.push({
+          itemType: ContextualMenuItemType.Header,
+          key: "private",
+          text: "Private Channels",
```

Comment:
@joaojmendes can you please update to use a loc string? It's fine if you can't do it and I will do it after the merge

Reply:
done